### PR TITLE
Add fix44 files

### PR DESCRIPTION
**Pull request by umitanuki**

### CODE REVIEW

**Review by neal — APPROVED** [review]
LGTM. is there no new enum/tags/fields need to be added? or you plan to add it later?

File: fix44/advertisement/Advertisement.generated.go [hunk 1]
```diff
@@ -0,0 +1,3304 @@
+package advertisement
```
Comment (neal) [inline fix44/advertisement/Advertisement.generated.go:1]: I realize this generated code is from the quickfix script but Go convention is to note that with a comment at the top of the file: https://golang.org/s/generatedcode

> To convey to humans and machine tools that code is generated, generated source should have a line that matches the following regular expression (in Go syntax):
>
> ```
> ^// Code generated .* DO NOT EDIT\.$
> ```
> This line must appear before the first non-comment, non-blank text in the file.

Maybe we should update the script to follow the latest convention and also send a PR to upstream.

**Comment by umitanuki** [comment]
> LGTM. is there no new enum/tags/fields need to be added? or you plan to add it later?

I suppose enum/tags/fields cover all versions

File: fix44/advertisement/Advertisement.generated.go (hunk 1 above)
Comment (umitanuki) [reply fix44/advertisement/Advertisement.generated.go:1]: This is pre-`go generate` era and this isn't generated by `go generate`. Not sure the convention in the doc applies to non-`go generate`

**Pull request merged**